Symfony/CS/Tokenizer/Tokens.php

```diff
@@ -929,6 +929,28 @@ public function isArrayMultiLine($index)
     }
 
     /**
+     * Check if there is an anonymous class under given index.
+     *
+     * @param int $index
+     *
+     * @return bool
+     */
+    public function isAnonymousClass($index)
+    {
+        $token = $this[$index];
+
+        if (!$token->isClassy()) {
+            throw new \LogicException('No classy token at given index');
```

Comment:
`isLambda` works like:
OK, I have a function and I want to check is it a lambda. So calling it on non-function token is providing an invalid argument, so an exception is rised.
So it's like `isThatFunctionALamba` but not `isThatTokenAFunctionThatIsLambdaFunctionAsWell`.

So here, to follow same principle, we should not allow to run method on interface / trait